networks/remote/ansible/roles/setup-validators/files/nodef.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=nodef
+Requires=network-online.target ee.service
+After=network-online.target
+
+[Service]
+Restart=on-failure
+User=nodef
+Group=nodef
+PermissionsStartOnly=true
+ExecStart=/usr/bin/nodef start > /home/nodef/.nodef/nodef.log
+ExecReload=/bin/kill -HUP $MAINPID
+KillSignal=SIGTERM
+
```

Comment:
you'd better add here `LimitNOFILE=4096` for preventing fd full.
```
Restart=on-failure
RestartSec=5
```
also recommended.

Reply:
add now